tdrs-backend/tdpservice/search_indexes/management/commands/seed_records.py

```diff
@@ -0,0 +1,135 @@
+"""Management command for seeding parsed records into the postgres db."""
```

Comment:
could you add a helper how to use this command? (options?)

Reply:
done!